torchtitan/experiments/deepseek_v3/run.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# torchrun --standalone --nproc-per-node 4 run.py
```

Comment:
is this file named 'run' instead of 'train' because it's not supposed to train?  It looks like it is using a training oriented pipeline-schedule (not fwd-only) but missing a loss function etc.  Maybe its just a temporary file for bootstrapping...

Reply:
It is a temp file. And it can diverge into inference and train later.